39.md

```diff
@@ -41,19 +41,23 @@ If a matching metadata event is found when trying to look up an identity, a clie
 ### `github`
 
 Identity: A GitHub username.  
-Proof: A GitHub Gist ID. This Gist should be created by `<identity>` with a single file that has the text `Verifying that I control the following Nostr public key: <npub encoded public key>`. This can be located at `https://gist.github.com/<identity>/<proof>`.
+Proof: A GitHub Gist ID. This Gist should be created by `<identity>` with a single file that has the text `Verifying that I control the following Nostr public key: <npub encoded public key>`.  
+This can be located at `https://gist.github.com/<identity>/<proof>`.
 
 ### `twitter`
 
 Identity: A Twitter username.  
-Proof: The ID of the Twitter tweet located at `https://twitter.com/<identity>/status/<proof>` with the text `Verifying my account on nostr My Public Key: "<npub encoded public key>"`
+Proof: A Tweet ID. The tweet should be posted by `<identity>` and have the text `Verifying my account on nostr My Public Key: "<npub encoded public key>"`.  
+This can be located at `https://twitter.com/<identity>/status/<proof>`.
 
 ### `mastodon`
 
 Identity: A Mastodon instance and username in the format `<instance>/@<username>`.
-Proof: A Mastodon post ID. This post should be published by `<username>@<instance>` and have the text `Verifying that I control the following Nostr public key: "<npub encoded public key>"`.
+Proof: A Mastodon post ID. This post should be published by `<username>@<instance>` and have the text `Verifying that I control the following Nostr public key: "<npub encoded public key>"`.  
+This can be located at `https://<identity>/<proof>`.
 
 ### `telegram`
 
 Identity: A Telegram user ID.
-Proof: A string in the format `<ref>/<id>` which points to a message published in the public channel or group with name `<ref>` and message ID `<id>`. This message should be sent by user ID `<identity>` and have the text `Verifying My Public Key: "<npub encoded public key>"`.
\ No newline at end of file
+Proof: A string in the format `<ref>/<id>` which points to a message published in the public channel or group with name `<ref>` and message ID `<id>`. This message should be sent by user ID `<identity>` and have the text `Verifying My Public Key: "<npub encoded public key>"`.  
```

Comment:
do we really need the string `Verifying that I control the following Nostr public key:` everytime? isn't nostr already implied by just using `<npub encoded public key>`? 
if we do need it, then it would be better if it's always the same and here it's different in this telegram block.

Reply:
it would be confusing for people on other social sites to see just a pubkey with no context. Having a grammatically correct statement is a good idea in my view. it's sort of a written contract/claim.

Reply:
Then still it makes sense to use the same sentence. I assume you accidentally had variations there.

Reply:
@Giszmo no, tweet and telegram text is different on purpose because of 16k+ tweets+tg posts that already have the text from nostr.directory. I think we should keep it to avoid having people do the same thing again. Or if we must keep the text the same then maybe we can put a note so clients can also accept variations on the wording of the statement.

Reply:
just and idea, but for github and similar sites a SSH pubkey could be used to more directly proof ownership. You could use openssl or smth to sign a message with the SSH identity that is visible on github (the pubkey can be found at a specific url. Similar for sr.ht etc)

This would then use the encoded signature as the third element of the tag array. Pubkeys are usually Ed25519 or RSA signatures, so clients would have to implement cryptography instead of using https+gist to verify validity.

This could be 'github+ssh:account', say.

Reply:
Just came back to say this comment above ^ is nonsensical; you would still need to get the file containing the pubkey and the. *additionally* do crypto. I'll leave this note here in case anyone else has this same brilliant idea haha.

gist + well known string = KISS